packages/eas-cli/src/commandUtils/easCommand.ts

```diff
@@ -0,0 +1,37 @@
+import { Command } from '@oclif/command';
+
+import { AnalyticsEvent, flushAsync, initAsync, logEvent } from '../analytics';
+import { getUserAsync } from '../user/User';
+import { ensureLoggedInAsync } from '../user/actions';
+
+export default abstract class EasCommand extends Command {
+  /**
+   * When user data is unavailable locally, determines if the command will
+   * request it from the backend
+   * @returns boolean - if authentication is required
+   */
+  requiresAuthentication(): boolean {
+    return true;
```

Comment:
defaults to 'yes prompt user for info and hit the backend if user data is not locally available'.

could also make the function abstract to force authors to implement + pay attention to it.